utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -1403,7 +1403,7 @@ def create_argument_parser():
 
     option('--android-arch', store,
            choices=['armv7', 'aarch64', 'x86_64'],
-           default='armv7',
+           default='aarch64',
```

Comment:
This seems unrelated? It's a welcome change, just seems not related to the test enablement.

Reply:
Unrelated to the title, yes, but these are all Android changes, and I specifically mention these non-test changes in the commit message.

Reply:
Sure, but it still helps to keep changes focused :)

Reply:
Right, this should probably be a separate patch, since it isn't just a test-only change.

Reply:
> Right, this should probably be a separate patch, since it isn't just a test-only change.

It never says it's a test-only change, the label used is `[android]` and the commit message lists the non-test changes.

If you would like, I can split this into separate commits, but I see no reason to go farther.

Reply:
The PR title doesn't mention any non-test changes, which makes it confusing. I initially assumed that this is a test-only change.
I think the arch change should be its own commit.

Reply:
OK, I will split this commit up.

Reply:
Done